File: src/main/clojure/pipelines/coma.clj
(ns pipelines.coma)

(use 'clojure.java.io)

(defn quitarComas[datos]
(def s (slurp datos))
(def sr (clojure.string/replace s "." ""))
(spit datos sr)
(def s (slurp datos))
(def sq (clojure.string/replace s "," ""))
(spit datos sq)
(def s (slurp datos))
(def sj (clojure.string/replace s ";" ","))
(spit datos sj)
)

File: src/main/java/coma.java

import java.io.IOException;

import org.openrdf.rio.RDFHandlerException;

import clojure.lang.RT;

public class coma {
	public static void main(String[] args) throws IOException, RDFHandlerException {
		RT.loadResourceScript("pipelines/coma.clj");
		 RT.var("pipelines.coma",
		 "quitarComas").invoke("./DatosIniciales/estacionesMetereologicas.csv");
	}
}
